packages/amplify-category-api/src/provider-utils/awscloudformation/utils/database-url.ts

```diff
@@ -0,0 +1,28 @@
+import { URL } from 'url';
+import { ImportedRDSType, RDSDataSourceConfig } from '@aws-amplify/graphql-transformer-core';
+
+export const parseDatabaseUrl = (databaseUrl: string): Partial<RDSDataSourceConfig> => {
```

Comment:
can we add a unit test for this method? What happens if I pass in any non-database URL?

Reply:
For now, we ignore the protocol as we only support mysql. I will add unit tests.

Reply:
Added unit tests and validation for url protocol.